assets/js/populateProducts.js

```diff
@@ -0,0 +1,59 @@
+// Import fetch
+const http = new Http();
+import { CarouselClass } from "./carouselClass.js";
+
+http.get("products").then((data) => {
+  populateCarousel(data);
+  populateProducts(data);
+});
+
+function populateCarousel(data) {
+  const carouselList = document.querySelector(".carousel-list");
+  const carouselContainer = document.querySelector("#carousel");
+  data.forEach((item, index) => {
+    const elementClass = index === 0 ? 'carousel-list__item carousel-list__item--active' : 'carousel-list__item';
+    if (item.isHighlighted) {
+      let newItem = `
+      <a href="" class="${elementClass}">
+        <figure class="carousel-list__image">
+          <img src="${item.imageUrl}" alt="Title">
+        </figure>
+        <div class="carousel-list__info">
+          <span>${item.title}</span>
+        </div>
+      </a>
+      `;
+      carouselList.innerHTML += newItem;
+    }
+  });
+
+  const carouselSlider = new CarouselClass(carouselContainer);
+  carouselSlider.init();
```

Comment:
Consider calling `init` from the `CarouselClass` constructor. Calling `.init` seperately like this make the `CarouselClass` harder to use.

Reply:
Have a question. Them I am calling init inside constructor, should I use it like `const carouselSlider = new Carousel(carouselContainer);
    carouselSlider;` or jus `new Carousel(carouselContainer);` ?